Adjust Dimensions, model and schema

annafralberts opened the pull request:
I have made an alternative schema + adjusted the fiscal model accordingly. 
Please do have a look @akariv

Commit:
Alternative Fiscal Model

I adjusted the fiscal model to allow to do better drill-downs and create a working hierarchies. But that does mean I "hacked" the model a bit. I moved from recipients/contracts to administrative and functional classification, although that is not a perfect fit.

Commit:
alternative fiscal schema

I tried to make a work around for the activity/recipient information as these need to be linked. I gave those two a functional classification, and the fund information the administrative classification. Also Geo-code and hierarchies proved to be difficult to map, again I tried a work-around, but it's not pretty.

akariv commented:
Hey @annafralberts - let's focus on `fiscal.schema.yaml` only (we're going to ditch the `fiscal.model.yaml` file pretty soon).
So if you could give it the proper name (i.e. no `alternative`) so I can see what actually was changed, that would be great.

Commit:
Changed dimensions of fiscal schema

I have modelled Functional Classifications for Beneficiary->Project->Details
For level three functional classification, I have added more than one column as they all give details regarding the project, is that even possible? If not, please put the others on unkown mapping and keep project description as level three.

Administrative Classification = Drill down for Fund-stream

Geo-classification is still difficult to create a good drill down, please advice.

annafralberts commented:
Done!

Commit:
Delete alternative

I have deleted this file, and adapted the changes here to fiscal schema file
